src/examples/snowy_wars.haml

```diff
@@ -46,21 +46,29 @@
                         <spawns>
                             <spawn team="red-team" kit="red">
                                 <point yaw="-45">
-                                    <cylinder base="-619.5,11,259.5" height="0" radius="2"/> <!-- West -->
+                                    <region>
+                                        <cylinder base="-619.5,11,259.5" height="0" radius="2"/> <!-- West -->
+                                    </region>
```

Comment:
This wouldn't be the correct way to fix this, the correct way would be to change the `<point ...>` to a `<region ...>` element.